game/src/assets/assets.json

```diff
@@ -2,10 +2,10 @@
   "image": [
     {
       "key": "sky",
```

Comment:
Minor comment, but since I'm here: maybe this should be called `title_background` or something like that.